packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -43,6 +49,8 @@ const metadata: Meta = {
 
 export default metadata;
 
+DesignToken.registerRoot(document.documentElement);
+
 const computedCSSValueFromTokenName = (tokenName: string): string => {
     return getComputedStyle(document.documentElement).getPropertyValue(
```

Comment:
I can't remember why we aren't anymore but wonder if we should just be using DesignToken.getValueFor: https://www.fast.design/docs/design-systems/design-tokens/#getting-values

Reply:
I'm not sure. Perhaps we were able to easily get all the token names from `design-token-names.ts` to generate CSS variable names but we weren't as easily able to get access to all the token objects from `design-tokens.ts`?